api/src/main/resources/liquibase-schema-only.xml

```diff
@@ -11,11 +11,9 @@
 
 -->
 <databaseChangeLog
-    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
-     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
-    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd
-    http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
```

Comment:
Sorry, this is the IDE that is doing the "magic" during the commit. I'm not sure how to prevent it. I did come across: https://github.com/suniala/openmrs-code-style-idea from the wiki. Again, I'm not sure if it will solve it.